### PR TITLE
chore: prometheus scrape format changes

### PR DESCRIPTION
**Pull request by sjungling**
clean-up unused dependencies

### CODE REVIEW

**Review by natedanner — APPROVED** [review]

File: src/main/java/io/moderne/organizations/OrganizationController.java [hunk 1]
```diff
@@ -0,0 +1,4 @@
+package io.moderne.organizations;
+
+public interface OrganizationController {
```
Comment (sjungling) [inline src/main/java/io/moderne/organizations/OrganizationController.java:3]: not sure if needed…

Comment (sjungling) [reply src/main/java/io/moderne/organizations/OrganizationController.java:3]: …and apparently it is…

**Comment by sjungling** [comment]
related to: https://github.com/moderneinc/moderne-organizations/issues/1

File: build.gradle.kts [hunk 2]
```diff
@@ -71,8 +70,11 @@ dependencies {
         exclude(module = "snakeyaml")
     }
 
-    implementation("io.micrometer:micrometer-core:1.+")
-    implementation("io.micrometer:micrometer-registry-prometheus:1.+")
+    implementation("io.micrometer:micrometer-core:latest.release")
```
Comment (jkschneider) [inline build.gradle.kts:73]: you can remove the core dependency

File: src/main/java/io/moderne/organizations/OrganizationMetricController.java [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+import io.micrometer.prometheus.PrometheusMeterRegistry;
+import io.prometheus.client.exporter.common.TextFormat;
+import lombok.Value;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+@RestController
+@Value
```
Comment (jkschneider) [inline src/main/java/io/moderne/organizations/OrganizationMetricController.java:11]: Can be a record

File: build.gradle.kts [hunk 4]
```diff
@@ -71,8 +70,11 @@ dependencies {
...
+    implementation("io.micrometer:micrometer-registry-prometheus:latest.release")
+
+    compileOnly("org.projectlombok:lombok:latest.release")
```
Comment (jkschneider) [inline build.gradle.kts:76]: Make controller a record and remove lombok

**Pull request merged**